pkg/controller/podautoscaler/horizontal.go

```diff
@@ -39,6 +39,8 @@ const (
 	// TODO: make it a flag or HPA spec element.
 	tolerance = 0.1
 
+	defaultTargetCPUUtilizationPercentage = 80
```

Comment:
If you have a better comment here I'm happy to change it. What I want to say here is that if a user doesn't specify any target here (as for now cpu, in the future other targets like custom metrics) this will be still a valid HPA object and HPA controller will scale based on some default policy, which may be a subject of changes in the future and is just an implementation detail (of course you can still take a look into the code).